src/datatypes/interval.coffee

```diff
@@ -165,6 +165,18 @@ module.exports.Interval = class Interval
     else # ol is null
       null
 
+  sameAs: (other, precision) ->
+    if ((@low == null and @high == null) or (other.low == null and other.high == null)) then return null
+    # pass in the opposite value to get the type since the checked value is null
+    else if @low == null then @low = minValueForInstance(@high)
```

Comment:
It looks like this and the next line need a test case to cover the case when @low == null or other.low == null.